examples/multi-gpu/multi-gpu.py

```diff
@@ -0,0 +1,74 @@
+"""
+==================
+Multi-GPU training
+==================
+
+``metatrain`` supports training a model with several GPUs, which can accelerate the
```

Comment:
It might be worth mentioning the type of parallelization employed and linking to the relevant PyTorch documentation?